### ISSUE TITLE
Problems with eSpeak NG in NVDA Next Snapshot Version next-15091,d89900a3

### ISSUE THREAD

**Issue by Living4God1991** [nvaccess/nvda#8231]
When using NVDA version next-15091,d89900a3, speech rate does not work how it should. At rate 100 without rate boost, NVDA speaks much slower than at rate 99. With rate boost, speech slows down above rate 18 and then seems to stay at the same rate regardless of where the slider is moved as long as you stay above rate 18. Words, especially those ending in vowel sounds, also sound as if they are cut off at the ends.
I noticed this first when I updated to this snapshot. During and after the update, speech seemed much slower than I'm used to. Once I noticed this, I tried changing the speech rate and found the results I mentioned above. To reproduce this, make sure you're using eSpeak NG and try changing the rate with and without rate boost. Pay particular attention to the rate values mentioned above.
I'm using NVDA version next-15091,d89900a3 on Windows 10 version 1803 build 17134.5. I also normally use the English America voice and the Klatt4 variant at rate 45 with rate boost turned on. I posted about this in the NVDA Skype group, and others seemed to be having the same issue when they looked into it.

**Comment by gregjozk** [comment]
confirmed.

**Comment by Brian1Gaff** [comment]
Hmm, I wonder if this the fix to stop speech overlap I see in their mail 
list not working as its supposed to?
 Sounds like once again ng has fixed one thing and broken something else.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by fastfinge** [comment]
I'm also having this issue with espeak-ng.  Reverting back to Speech Player in Espeak for the time being.

**Comment by rhdunn** [comment]
Thanks for locating the cause of the regression. I have reverted that commit.

**Comment by michaelDCurran** [comment]
Fixed in espeak, and now in NVDA next.
